frontend/styles/components/calendar/desktop/CompactCalendarSidebar.module.scss

```
@@ -1,4 +1,4 @@
-@import '../../Variables.module.scss';
+@import '../../../Variables.module.scss';
```

Stylelint (17.14.0)

[error] Unexpected extension ".scss" in `@import` (scss/load-partial-extension)

(scss/load-partial-extension)